FromAndriy

```diff
@@ -1,3 +1,2 @@
-Test file
```

Comment:
Can I remove this line? @VasylNaulychnyi

Reply:
@AndriiNaulychnyi Yes, you can.